### DISCUSSION TITLE
Best Practice Guide?

### DISCUSSION THREAD

**Discussion by s-l-w** [apohl79/audiogridder#1140] [General]
Hello apohl79 & community,

Congratulations & thank you for building audiogridder and making it available for free (and via MIT). Its truly easy to use and works excellent so far. I am currently still testing different setups and workflows.

1) I am wondering if something like a best practice guide exists? When I browse through the questions here there seems to be a lot of confusion about the ideal setup (from trying wifi to very old hardware, others some not being able to set up VSTs correcty).

2) Any cool examples of audiogrid in production (written down or youtube) for larger setups and how they work, i.e. multiple hosts (does it have advantages beyong CPU bandwidth?)?

Again - thanks for making this. And sorry if I missed 1), just reply RTFM ;-)

**Comment by emack999** [comment]
A 'best practice guide' is unlikely, as it is still in some early stages.  Also, there is no 'ideal setup' because it can be used in many ways---cross-OS configurations, multiple machines, local mode etc..  It's up to us to find the ways that work well for our particular use-case, and perhaps share a little bit of what we find useful.  Also, not all plugins play well with AG, so there's some of that as well.

PS. Don't use WiFI! :)

**Comment by MM502** [comment]
@rncwalker left a very useful tip in one of my threads:
https://github.com/apohl79/audiogridder/discussions/1094#discussioncomment-4124886

Other than that it's trial and error at this point. I have a powerful CPU and am currently only offloading Acustica Audio plugins. I experimented briefly with VIs but it's neither been successful for me nor has it been necessary to do so. I do run large projects with a lot of outboard gear and AG is brilliant for those plugins that have a huge CPU penalty.

Other than that I have a dedicated 10Gbe card in the server and client and they are directly connected. This is not necessary but as I was system building I decided to put them in so there would be no potential clashed with other network traffic.
